src/Microsoft.Health.Fhir.Core/Features/Persistence/Orchestration/BundleOrchestratorOperation.cs

```diff
@@ -221,7 +221,7 @@ public void Cancel(string reason)
         {
             try
             {
-                _logger.LogWarning("Bundle Operation {Id}. Bundle Orchestrator Operation was requested to cancel. Reason: {Reason}", Id, reason);
+                _logger.LogWarning($"Bundle Operation '{Id}'. Bundle Orchestrator Operation was requested to cancel. Reason: {reason}");
```

Comment:
Can we keep the variables as a list of parameters instead of string injections? It makes summarizing logs easier if the body has consistent values.